### ISSUE TITLE
Resource identifier should be placed as a path parameter

### ISSUE THREAD

**Issue by wypiorsebastian** [aspnetrun/run-aspnetcore-microservices#25]
I know it's not a course about RESTful principles but maybe you'll consider in the future extracting resource id from request body to path parameter ?

https://github.com/aspnetrun/run-aspnetcore-microservices/blob/c30712aa099bc335f5613a3b196ea1e943f800a5/src/Services/Catalog/Catalog.API/Controllers/CatalogController.cs#L85

**Comment by mehmetozkaya** [comment]
yes you exactly right, this is good to fix in this issue. I will check it later thanks.

**Comment by wypiorsebastian** [comment]
No problem. Glad I could help. BTW - great course :)
